classes/payment/PaymentManager.php

```diff
@@ -96,10 +92,10 @@ public function isConfigured()
     public function getPaymentForm($queuedPayment)
     {
         $paymentPlugin = $this->getPaymentPlugin();
-        if ($paymentPlugin !== null && $paymentPlugin->isConfigured($this->_context)) {
-            return $paymentPlugin->getPaymentForm($this->_context, $queuedPayment);
+        if (!$paymentPlugin?->isConfigured($this->_context)) {
+            throw new Exception('Payment not configured');
```

Comment:
Was there a friendly user-facing error than the default exception handler prior to this change?

Reply:
AFAICR due to the lack of `null` (`false`) checks, it was just going to fail with a fatal error:
```php
$paymentForm = $paymentManager->getPaymentForm($queuedPayment);
$paymentForm->display($this->request);
```

Reply:
If I recall the payment code generally checks the configured status before calling functions like this in the first place, so any error handling would have already happened. But neither of us is sure, and the payment aspects of the software already went through the community testing process and are a little finicky to set up for testing, so I'd rather leave this for now.

Reply:
No problem! I can move this, together with the other things that I didn't push, to the post-3.4 era.